transform/app/api/routes/solr_utils/solr.py

```diff
@@ -0,0 +1,65 @@
+from fastapi import APIRouter, Query
+from datetime import datetime
+import requests
+
+from app.transform.utils.loader import load_env_vars
+from app.transform.schemas.properties.env import (
+    SOLR_ADDRESS,
+    SOLR_PORT,
+    SOLR_COLLECTION_NAMES,
+)
+
+env_vars = load_env_vars()
+router = APIRouter()
+
+
+@router.post("/create_collections")
+async def create_solr_collections(
+    all_collection_config: str = Query(
+        ...,
+        description="Config name for 'all_collection'",
+        example="all_collection_oag56_v203",
+    ),
+    provider_config: str = Query(
+        ..., description="Config name for 'provider'", example="provider_v101"
+    ),
+    collection_prefix: str = Query(
+        "",
+        description="Prefix for collection names. "
+        "It is recommended to use convention 'oag<dump_version>'",
+        example="oag56",
+    ),
+    date: str = Query(
+        None,
+        description="Date string in the format 'YYYYMMDD'. "
+        "Defaults to the current date if not provided.",
+        example="20231114",
```

Comment:
we shouldnt have an example here. Why? Because 99% of the time we just want a current date, therefore we would be forced to delete the default. It will be better to just get rid of an example because its valid only today

Reply:
You're right, I assumed it worked differently. Corrected.